src/source/raster_dem_tile_source.js

```diff
@@ -74,7 +73,7 @@ class RasterDEMTileSource extends RasterTileSource implements Source {
             }
 
             if (serialized) {
-                tile.dem = (deserialize(serialized): any);
+                tile.dem = serialized;
```

Comment:
Rename `serialized` to `dem` or `demData`, since `done()` will now be called with the object already deserialized.